src/index.html

```diff
@@ -11,6 +11,59 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <!-- Large search form -->
+
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+      class="large__form"
+    >
+      <div class="container large__container">
+        <button
+          type="submit"
+          class="button large__button"
+        >
+          <img
+            src="./images/Search.svg"
+            alt="search__image"
+            class="large__image"
+          >
+        </button>
```

Comment:
It shouldn't be button, it just an decoration image

Reply:
Also this button redirects me to page with error and it's confusing
<img width="1418" alt="Screenshot 2023-01-24 at 18 47 20" src="https://user-images.githubusercontent.com/84275496/214355288-150883a5-bc5b-4976-9370-bd341c8d5c66.png">